feat: Add Postgresql Service (#1)
dusdjhyeon/terraform-provider-ncloud: docs/data-sources/postgresql_products.md

---
subcategory: "PostgreSQL"
---

# Data Soruce: ncloud_postgresql_products

Get a list of PostgreSQL products.

~> **NOTE:** This only supports VPC environment.

## Example Usage

```terraform
data "ncloud_postgresql_products" "all" {

>     image_product_code = "SW.VPGSL.OS.LNX64.CNTOS.0708.PGSQL.133.B050"

Reviewer: "SW.VPGSL.OS.LNX64.ROCKY.0810.PGSQL.B050" 로 변경

    filter {
        name = "product_type" 
        values = ["STAND"]
    }

    output_file = products.json"
}

output "product_list" {
    value = {
        for product in data.ncloud_postgresql_products.all.product.list :
        image.product_name => image.product_coe
    }
}
```

Outputs:
`terraform
list_image = {

>     "vCPU 2EA, Memory 8G" = "SVR.VSVR.STAND.C002.M008.NET.SSD.B050.G002"

Reviewer: Postgresql 의 실제 응답 코드값으로 변경
SVR.VPGSL.DBSVR.STAND.C002.M008.NET.SSD.B050.G002

}

## Argument Reference

The following arguments are supported:

> * `image_product_code` - (Required) Youc an get one from `data.ncloud_postgresql_image_products`, This is a required value, and each available PostgreSQL's specification varies depending on the PostgreSQL image product.

Reviewer 2: ```suggestion
* `image_product_code` - (Required) You can get one from `data.ncloud_postgresql_image_products`, This is a required value, and each available PostgreSQL's specification varies depending on the PostgreSQL image product.
```
typo
